### DISCUSSION TITLE
Is Kimai free to use for On-Premise deployment?

### DISCUSSION THREAD

**Discussion by birkanthedev** [kimai/kimai#4969] [Questions & Answers]
Hey, guys. I need a free, on-premise time tracker. On kimai's website, there are pricing plans available for cloud customers. Does this apply to on premise customers as well?  Or is it completely free for on-premise deployment?

**Comment by niels-przybilla** [comment]
Hi @birkanthedev ,

yes. You can install it on-premise from the github repository and you do not need to buy a license

Best wishes

NIiels
